docs/source/glossaries.rst

```diff
@@ -95,6 +100,7 @@ UD          Unreliable Datagram (InfiniBand)
 uGNI        user level generic network interface(Cray)
 UMR         User mode memory registration
 VPI         Virtual Protocol Interconnect
+WFE         Wait for event
 WIP-DNM     Work-in-Progress-Do-Not-Merge
 WQ          Work Queue(InfiniBand)
```

Comment:
add whitespace between `Queue` and `(InfiniBand)`

pls, fix other places too
thank you

Reply:
done. fix other places too.